lss-manager-v3/css/main.css

```diff
@@ -404,7 +404,18 @@ right: 14.5px;
   background-color: #009dff !important
 }
 
-
+.aao-counter {
```

Comment:
Ich weiß, dass es hierzu Diskussionen gab.
Dennoch würde ich sagen, dass Modul-spezifischer Code nichts in einer globalen css Datei zu suchen hat. Das widerspricht dem Konzept.

Reply:
Ja bin ich auch der Meinung. Hab wahrscheinlich irgendwo mal gesagt das es ok wäre in der main - soll aber natürlich nur Komponenten basiert sein.